src/fragments/lib-v1/restapi/native_common/getting-started/common.mdx

```diff
@@ -61,7 +61,7 @@ import flutter11 from "/src/fragments/lib-v1/restapi/flutter/getting-started/30_
 
 ## Make a POST Request
 
-Copy and paste the following code in your application so that it runs only once when the app starts:
+Send a POST request with a JSON body.
```

Comment:
This affects all the platforms, is it designed to be that way?

Reply:
Yes, left a callout comment about this but I found it confusing and thought maybe it's a mistake, so changed it here. (Not sure why someone would execute a REST API call only when the app starts). If I'm wrong happy to change back.

Reply:
Also, just realized I left that comment in the part that is not for v1 but same thing applies.